#source/scss/reset.scss

```diff
@@ -0,0 +1,36 @@
+*{
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+*,*:before,*:after{
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+:focus,:active{outline: none;}
```

Comment:
I removed the outline so that there were no borders:
![without outline](https://user-images.githubusercontent.com/54203405/118121115-9e5c2680-b3f9-11eb-9ea4-29bbde2561fe.jpg)

With outline it looks like this:
![with outline](https://user-images.githubusercontent.com/54203405/118121146-af0c9c80-b3f9-11eb-96a3-f66b1f52b0a3.jpg)